OpenRA.Mods.Common/Projectiles/InstantHit.cs

```diff
@@ -71,7 +71,11 @@ public void Tick(World world)
 
 			if (!doneDamage)
 			{
-				args.Weapon.Impact(Target.FromPos(target), args.SourceActor, args.DamageModifiers);
+				if (info.Inaccuracy.Length > 0)
```

Comment:
See https://github.com/pchote/OpenRA/commit/1cd2eab90db59496317f11cc7ad5d8d71182df65 for what I mean (note: I haven't tested it).